README.quickstart.vscode.md

```diff
@@ -91,14 +91,15 @@ contextual documentation, as shown below.
 
 ### Supported runtimes
 
-The Toolkit _local SAM debugging_ feature supports these runtimes:
+The Toolkit _local SAM debugging_ feature supports these Lambda runtimes:
 
 -   C# (.NET Core 2.1, 3.1; .NET 5.0)
+-   Go (1.x)
 -   Java (8, 8.al2, 11)
 -   JavaScript (Node.js 10.x, 12.x, 14.x)
 -   Python (2.7, 3.6, 3.7, 3.8)
 
-For more information see [Working with AWS Serverless Applications](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/serverless-apps.html) in the user guide.
+For more information see [Working with AWS Serverless Applications](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/serverless-apps.html) in the user guide. To get setup for local debugging, see [Configuring your toolchain](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/setup-toolchain.html).
```

Comment:
it's also mentioned in `#setup` in this doc doc https://github.com/aws/aws-toolkit-vscode/blob/7f3b9298adc95f2c996fc3dc69856c7d3f4c2139/README.quickstart.vscode.md#setup